.storybook/decorators/context.js

```diff
@@ -134,6 +134,11 @@ export const withContextWrapper = makeDecorator({
 				if (hasStaticElement && staticKey && staticColorSettings[staticKey]) {
 					container.style.background = staticColorSettings[staticKey].background;
 				}
+
+				// Demonstrate the "Modal" component with a different darker background, so its bounds are visible.
+				if (rootClass == "spectrum-Modal") {
+					container.style.background = "var(--spectrum-transparent-black-400)";
+				}
```

Comment:
I am really not sure about including a setting in the decorator that's specific to only one component and likely something we only want in test mode.

Reply:
This helps things for the user, not just for test mode. Before it was unclear where the boundaries were, especially when changing the variant control. Example:
- https://opensource.adobe.com/spectrum-css/preview/?path=/story/components-modal--default&args=variant:fullscreen vs 
-  https://pr-2854--spectrum-css.netlify.app/preview/?path=/story/components-modal--default&args=variant:fullscreen

I don't like the idea of a component specific exception either.
1. Should I remove it and just let it display as-is?
2. What about doing a `customStyles` with `--mod-modal-background-color: var(--spectrum-background-layer-2-color);` instead to give the modal a contrasting color?
3. Something else?

Reply:
I'll let Cassondra make the final decision here, but from the options you mentioned, maybe the `customStyles` is the way to go. Is that more in line with how we're using `customStyles` in other components?

Reply:
I just wanted to note that the `customStyles` option would be changing the opposite of what's on the PR now (component foreground instead of story body background). Which may not be desirable.

Reply:
I've dropped that commit and have addressed this by displaying the Underlay component beside the Modal, similar to Dialog. I've also added a `showUnderlay` control.